test/benchmark/locations.py

```diff
@@ -1,3 +1,6 @@
+from ..general import gen_steps
```

Comment:
This breaks, you are meant to be able to run the benchmark directly, which means there is no parent

Reply:
Oh true, good catch